cmd/integrationArtifactGetMplStatus.go

```diff
@@ -106,8 +106,20 @@ func runIntegrationArtifactGetMplStatus(
 		if parsingErr != nil {
 			return errors.Wrapf(parsingErr, "HTTP response body could not be parsed as JSON: %v", string(bodyText))
 		}
-		mplStatus := jsonResponse.Path("d.results.0.Status").Data().(string)
-		commonPipelineEnvironment.custom.iFlowMplStatus = mplStatus
+		if jsonResponse.Exists("d", "results", "0") {
+			mplStatus := jsonResponse.Path("d.results.0.Status").Data().(string)
+			commonPipelineEnvironment.custom.iFlowMplStatus = mplStatus
+
+			mplId := jsonResponse.Path("d.results.0.MessageGuid").Data().(string)
```

Comment:
How about changing the name, as suggested, to mplID?

Reply:
done